packages/appstore/src/modules/onboarding/onboarding.tsx

```diff
@@ -46,6 +47,20 @@ const Onboarding = ({ contents = trading_hub_contents }: TOnboardingProps) => {
 
     const onboarding_step = number_of_steps[step - 1];
 
+    const footer_header = contents[onboarding_step]?.footer_header;
+    const footer_text = contents[onboarding_step]?.footer_text;
+
+    const is_eu_footer_header = 'CFDs or Multipliers';
+    const is_eu_footer_text = 'You can choose between CFD trading accounts and Multipliers accounts';
+
+    const footer_header_text =
```

Comment:
you need to change this content for this specific step based on is_eu condition in the `packages/appstore/src/constants/trading-hub-content.tsx` file

Reply:
const footer_desctiption = is_eu
            ? eu_footer_text
            : footer_text;